java-spiffe-helper/example.conf

```diff
@@ -0,0 +1,12 @@
+keyStorePath = /tmp/keystore.p12
+keyStorePass = example123
+keyPass = pass123
+
+trustStorePath = /tmp/truststore.p12
+trustStorePass = otherpass123
+
+keyStoreType = pkcs12
+
+keyAlias = spiffe
+
+spiffeSocketPath = unix:/tmp/agent.sock
```

Comment:
This file is redundant now as #199  is merged, which adds a `java-spiffe-helper.properties`, an example configuration in the default config path. That file could be copied in the Dockerfile. WDYT?